v2/kafka-to-bigquery/src/test/java/com/google/cloud/teleport/v2/templates/KafkaToBigQueryFlexJsonIT.java

```diff
@@ -90,6 +90,7 @@ public void testKafkaToBigQuery() throws IOException {
   }
 
   @Test
+  @TemplateIntegrationTest(value = KafkaToBigQueryFlex.class, template = "Kafka_to_BigQuery_Flex")
```

Comment:
It looks like this test only verifies that good records were written to the main BQ table but doesn't verify that the bad records were written to the DLQ table.

Reply:
Yes. That is the intention of that test. For the DLQ case, we have a separate test called "testKafkaToBigQueryWithExistingDLQ"

Reply:
Added extra check for the DLQ table